sycl/source/detail/scheduler/scheduler.cpp

```diff
@@ -141,6 +141,19 @@ EventImplPtr Scheduler::addHostAccessor(Requirement *Req) {
 
 void Scheduler::releaseHostAccessor(Requirement *Req) {
   Req->MBlockedCmd->MCanEnqueue = true;
+  MemObjRecord* Record = Req->MSYCLMemObj->MRecord.get();
+  for (Command *Cmd : Record->MReadLeafs) {
```

Comment:
`Record->MReadLeaves` ?

Reply:
You are right, but to fix that I have to change the name of the variable in whole library.

Reply:
Yes, please separate refactoring from functional changes.

Reply:
Sure thing, will do with another patch.